dev/com.ibm.ws.jdbc_fat/test-applications/basicfat/resources/WEB-INF/ibm-web-bnd.xml

```diff
@@ -5,6 +5,10 @@
   <resource-ref name="jdbc/dsfat3" binding-name="jdbc/dsfat3">
     <authentication-alias name="auth1"/>
   </resource-ref>
+  
+  <resource-ref name="jdbc/dsfat5ref1" binding-name="jdbc/dsfat5">
+    <authentication-alias name="derbyAuth1"/>
+  </resource-ref>
 
```

Comment:
What was behind moving derbyAuth1 from the datasource in server.xml to the resource reference in ibm-web-bnd.xml?  Is that part of the move to creating the table during init?

Reply:
So the reason I did this was so that I could use `dsfat5` in the `testConfigChangeAuthData` test instead of dsfat6.  The main difference between the two, and why I wanted to change, was that dsfat6 uses connection pooling while dsfat5 does not.  Our continuous tests were experiencing an intermittent failure where the server was attempting to shutdown the connection pool just as dsfat6 was trying to get a connection for this test.  Since this test has nothing to do with connection pooling, I figured we could avoid this intermittent failure by using a datasource that does not use connection pooling. Defect: 263440

Reply:
Thanks and good thinking!